### PR TITLE
Repository tidy-up

### LINKED ISSUE #832: Bootstrap new codebase
Check existing migrations and data bootstrapping scripts are still relevant. Update and remove where necessary.

- [ ] Bootstrap database tables
    - [ ] Squash existing migrations
- [ ] Bootstrap initial data
    - [x] anon user
    - [ ] ?
- [x] Update README where necessary
- [x] Remove old admin scripts

### PR DESCRIPTION
**Pull request by brew**
This PR removes some old maintenance scripts and unused project resources.

Closes #832.

### CODE REVIEW

File: fixtures/utils.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+const userIds = [
   uuid.v4(),
   uuid.v4(),
   uuid.v4(),
   uuid.v4()
 ];
 
 module.exports = {
-  userIds: userIds
+  userIds: userIds,
```
Comment (nightsh) [inline fixtures/utils.js]: You can use the new [object literals syntax](http://www.benmvp.com/learning-es6-enhanced-object-literals/):

```js
module.exports = {
  userIds,
  ANONYMOUS_USER_ID: controllerUtils.ANONYMOUS_USER_ID
}
```

File: scripts/utils.js [hunk 2]
```diff
@@ -17,35 +17,5 @@ var loadData = function(promises) {
...
-    'yes': true,
-    'no': false,
-    'unsure': null
-  },
-  idMapper: {
-    'national': 'global',
-    'aus-region': 'australia',
-    'car-region': 'caribbean'
-  }
+  loadData: loadData
```
Comment (nightsh) [inline scripts/utils.js]: Same notation as in prev comment applies here.

File: README.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ Now we should be ready to run the server:
 Other things you can do:
 
 * Run the test suite with `npm test`
-* Check your code style with npm run jscs (according to the Google style guide)
+* Check your code style with `node_modules/eslint/bin/eslint.js census/<filename.js>` using the eslint config `/.eslintrc`.
```
Comment (nightsh) [inline README.md]: You can add a new `script` to `package.json` doing this on all the .js files in the project. For `eslint`, you can have an [ignore](http://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories) file to specify what should not be included in the linting process. Might save you a few typos or when you forget to lint something.

Also, if you add a `npm run lint` command as described above, you can [add it into the `posttest` section](http://blog.npmjs.org/post/127671403050/testing-and-deploying-with-ordered-npm-run-scripts) (see section *"Run scripts after testing"*), so linter runs automatically if all the tests are green. This way you won't be bothered with fixing linting issues until the code actually works.

Comment (brew) [inline README.md]: The codebase is currently in transition, so not all files will pass linting. Now might be a good time to go through and align all files with the linting standards, then I'd be more comfortable with adding an `npm run` command and posttest. I like the idea of only checking linting if all tests pass though.

Comment (brew) [inline README.md]: @nightsh took your advice and brought all files in line with current linting rules, and set up the `posttest` hook. https://github.com/okfn/opendatasurvey/pull/894

**Pull request merged**